### DISCUSSION TITLE
could not create a primitive descriptor for a reorder primitive

### DISCUSSION THREAD

**Discussion by 704572066** [PaddlePaddle/PaddleOCR#12781] [Q&A]
paddleocr 进行识别的时候 
ocr = PaddleOCR(use_angle_cls=True, lang=language, enable_mkldnn=True)  
lang选择ch，开启enable_mkldnn=true，如果图片有中文会报错：could not create a primitive descriptor for a reorder primitive
请提供下述完整信息以便快速定位问题/Please provide the following information to quickly locate the problem

- 系统环境/System Environment：centos7.5
- 版本号/Version：Paddle：2.4.0  PaddleOCR：2.7 问题相关组件/Related components：
- 运行指令/Command Code：
- 完整报错/Complete Error Message：    texts = text_ocr_v4(img, language)[0]
            │           │    └ 'ch'
            │           └ <PIL.Image.Image image mode=RGB size=1200x1600 at 0x7F8BEC9BBAC0>
            └ <function text_ocr_v4 at 0x7f8c7f638ee0>

  File "/home/gujie/PaddleWebOCR/paddlewebocr/pkg/ocr.py", line 39, in text_ocr_v4
    return ocr.ocr(np.array(img), cls=True)
           │   │   │  │     └ <PIL.Image.Image image mode=RGB size=1200x1600 at 0x7F8BEC9BBAC0>
           │   │   │  └ <built-in function array>
           │   │   └ <module 'numpy' from '/root/.virtualenvs/ocr_venv_3.8/lib/python3.8/site-packages/numpy/__init__.py'>
           │   └ <function PaddleOCR.ocr at 0x7f8c7f638ca0>
           └ <paddleocr.paddleocr.PaddleOCR object at 0x7f8bec9bbb50>

  File "/root/.virtualenvs/ocr_venv_3.8/lib/python3.8/site-packages/paddleocr/paddleocr.py", line 645, in ocr
    dt_boxes, rec_res, _ = self.__call__(img, cls)
                           │    │        │    └ True
                           │    │        └ array([[[ 37,  35,  36],
                           │    │                  [ 37,  35,  36],
                           │    │                  [ 37,  35,  35],
                           │    │                  ...,
                           │    │                  [186, 175, 174],
                           │    │                  [184...
                           │    └ <function TextSystem.__call__ at 0x7f8c7f615790>
                           └ <paddleocr.paddleocr.PaddleOCR object at 0x7f8bec9bbb50>
  File "/root/.virtualenvs/ocr_venv_3.8/lib/python3.8/site-packages/paddleocr/tools/infer/predict_system.py", line 95, in __call__
    rec_res, elapse = self.text_recognizer(img_crop_list)
                      │    │               └ [array([[[227, 211,  27],
                      │    │                         [227, 211,  27],
                      │    │                         [226, 211,  27],
                      │    │                         ...,
                      │    │                         [230, 215,  27],
                      │    │                         [23...
                      │    └ <tools.infer.predict_rec.TextRecognizer object at 0x7f8c4920f5e0>
                      └ <paddleocr.paddleocr.PaddleOCR object at 0x7f8bec9bbb50>
  File "/root/.virtualenvs/ocr_venv_3.8/lib/python3.8/site-packages/paddleocr/tools/infer/predict_rec.py", line 616, in __call__
    self.predictor.run()
    │    │         └ <instancemethod run at 0x7f8ca36712e0>
    │    └ <paddle.fluid.libpaddle.PaddleInferPredictor object at 0x7f8bec98be30>
    └ <tools.infer.predict_rec.TextRecognizer object at 0x7f8c4920f5e0>

RuntimeError: could not create a primitive descriptor for a reorder primitive

我们提供了AceIssueSolver来帮助你解答问题，你是否想要它来解答(请填写yes/no)?/We provide AceIssueSolver to solve issues, do you want it? (Please write yes/no):

请尽量不要包含图片在问题中/Please try to not include the image in the issue.

**Comment by cxiaowen** [comment]
这个问题解决了没，怎么解决的？
